cdap-watchdog-api/src/main/java/io/cdap/cdap/api/metrics/MetricsWriterContext.java

```diff
@@ -22,6 +22,12 @@
  * Context passed to {@link MetricsWriter#initialize(MetricsWriterContext)}.
  */
 public interface MetricsWriterContext {
+
+  /**
+   * CDAP version label
+   */
+  String CDAP_VERSION = "cdap.version";
```

Comment:
nit: we have one for apps
 public static final String APP_CDAP_VERSION = "app.cdap.version";

Not sure if we can consolidate them into one.

Reply:
APP_CDAP_VERSION is used to refer to application specific CDAP version.